job-search-react/src/components/JobApp/Card/Card.js

```diff
@@ -2,11 +2,31 @@ import React from 'react';
 
 export default function Card({ isSubmitted, jobData }) {
 	return (
-		<pre
-			className="grid-container"
-			style={{ display: isSubmitted ? 'block' : 'none' }}
-		>
-			{JSON.stringify(jobData, null, 4)}
-		</pre>
+		<div class="grid-container">
+			<div class="grid-item grid-item-1">{jobData.title}</div>
+			<div class="grid-item grid-item-2">
+				Company: {jobData.company.display_name} -{' '}
+				{jobData.location.display_name}
+			</div>
+			<div class="grid-item grid-item-3">Remote: {}</div> {/* TODO!!! */}
+			<div class="grid-item grid-item-4">
+				Job Description: {/* this regex removes html tags from descriptions */}
+				{jobData.description.replace(
+					/<[^>]*(>|$)|&nbsp;|&zwnj;|&raquo;|&laquo;|&gt;/g,
```

Comment:
Why do we need this regex? I see that it is to remove html tags. (why remove them?)
https://reactjs.org/docs/dom-elements.html#dangerouslysetinnerhtml